src/style.css

```diff
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+/* stylings for navigation */
+nav {
+  position: absolute;
+  height: 10%;
+  width: 100%;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo-container {
+  display: flex;
+  align-items: center;
+  padding-top: 2.6%;
+  padding-left: 3%;
+  color: #fff;
+  background-color: #3c3a39;
+}
+
+#logo,
+#ham {
+  background-color: #3c3a39;
+  color: #fff;
+}
+
+#logo {
+  font-family: 'Roboto', sans-serif;
+  color: white;
+}
+
+.hamberger-btn {
+  background-color: #3c3a39;
+  display: block;
+  font-size: 25px;
+  color: white;
+  float: right;
+  line-height: 80px;
+  margin-right: 40px;
+  cursor: pointer;
+}
+
+.section {
+  background-color: #1c1a19;
+  height: 100%;
+}
+
+.container {
+  position: relative;
+  background: url(Img/illustration-header-mobile.png);
+  background-repeat: no-repeat;
+
+  /* background-position: right bottom; */
+  background-position-y: 185px;
+  background-attachment: fixed;
+  background-size: 25%;
+  background-position: right bottom;
```

Comment:
- [ ] [OPTIONAL] Excellent job including your background image in your CSS file 🚀. However, please kindly consider positioning the image in the right position to stick as much as possible to the Figma design 🤓.

This is the Figma design 👇🏽


![figma](https://user-images.githubusercontent.com/116701485/235282729-0618ebd1-3fdc-4228-a96a-d83449efb5f6.png)

This is your design 👇🏽 (almost there 😅)

![design](https://user-images.githubusercontent.com/116701485/235282736-7079219a-61fb-4fa7-8b97-159fae7adcf8.png)